packages/orbit-components/src/Drawer/index.tsx

```diff
@@ -243,17 +142,19 @@ const Drawer = ({
               </Stack>
             )}
             {onClose && <DrawerClose onClick={onClose} ref={closeButtonRef} title={labelHide} />}
-          </StyledDrawerHeader>
+          </div>
         )}
-        <StyledDrawerContent
-          noPadding={noPadding}
-          hasClose={!!onClose}
-          hasTopPadding={!!(title || actions)}
+        <div
+          className={cx(
+            !!onClose && noPadding && "mt-lg",
```

Comment:
This used to be `!hasClose && noPadding`, but `hasClose = !!onClose`, so in here it should only be `!onClose && noPadding`, no?